css-two-column-layout/styles.css

```diff
@@ -0,0 +1,73 @@
+* {
+  font-family: Karla, sans-serif;
+}
+
+.container {
+  max-width: 800px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.row {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+
+.row-inner {
+  display: flex;
+}
+
+.column-full {
+  text-align: center;
+  margin-top: 2rem;
+}
+
+.column-fourths {
+  width: 400px;
+  height: 200px;
+  flex-basis: 25%;
+  padding: 0.4rem;
+}
+
+.column-three-fourth {
+  flex-basis: 75%;
+  padding: 1rem;
+}
```

Comment:
Only `flex-basis` should be applied to `column-` classes. Anything else should be on a separate rule.